types/redux-thunk/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
```

Comment:
Why make private? Is this so that you can avoid publishing until `redux-thunk@2.3` releases?